src/Mod/BIM/importers/importSH3DHelper.py

```diff
@@ -933,9 +933,9 @@ def get_wall_spine(self, wall):
     def get_faces(self, wall):
         """Returns the name of the left and right face for `wall`
 
-        The face names are suitable for selection later on when creating 
+        The face names are suitable for selection later on when creating
         the Facebinders and baseboards. Note, that this must be executed
-        once the wall has been completly been constructued. If a window
+        once the wall has completely been constructed. If a window
```

Comment:
Unsure of `been` should be before or after

Reply:
[fb2823d](https://github.com/FreeCAD/FreeCAD/pull/20173/commits/fb2823d2bc0ef3c50c16d7467480936618747eb7)